xml/acl-render.xsl

```diff
@@ -0,0 +1,225 @@
+<!--
+ Copyright 2019 the Pacemaker project contributors
+
+ The version control history for this file may have further details.
+
+ Licensed under the GNU General Public License version 2 or later (GPLv2+).
+ -->
```

Comment:
As we start getting more unrelated components under xml, I'd like to start using subdirectories for logical purposes (eventually I'd like to move all the CIB XML RNGs/XSLTs to xml/cib). Let's put this in a new subdirectory.

Reply:
Resolved with enumerated "namespaces" as mentioned via other channel.